packages/mongodb-rag-ingest/src/commands/embed.ts

```diff
@@ -5,38 +5,62 @@ import {
   withConfig,
   withConfigOptions,
 } from "../withConfig";
-import { updateEmbeddedContent } from "mongodb-rag-core";
+import { logger, updateEmbeddedContent } from "mongodb-rag-core";
 
 type EmbeddedContentCommandArgs = {
```

Comment:
i think these args should be assigned to the `embed update` command, not kept at thte command root lvl